README.adoc

```diff
@@ -134,6 +135,7 @@ The `handles()` method inspects the HTTP response code to determine whether an e
 == Injecting the client with dependency injection
 
 Now, instantiate the `SystemClient` interface and use it in the `inventory` service. If you want to connect only with the default host name, you can easily instantiate the `SystemClient` with CDI annotations.
+CDI injection makes things a lot simpler when it comes to bootstrapping the client.
```

Comment:
"Things" sounds ambiguous to me. What about..."CDI injection simplifies the process of bootstrapping the client."?